css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,51 @@
+@import "../page-vars";
+
+.lune .ampstart-headerbar {
+  display: flex;
+  align-items: center;
+
+  height: var(--layout-header-height-m);
+
+  box-shadow: none;
+
+  will-change: transform;
+}
+
+.lune .ampstart-headerbar-title {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  font-size: 1.3125rem; /* 21px */
+  font-weight: var(--font-weight-black);
```

Comment:
Black is a font weight thicker than bold and doesn't relate to a theme. We can refactor to --font-weight-900 if that makes more sense?